### PR TITLE
Add parameter for reflow on focus mixin 

### PR DESCRIPTION
**Pull request by ricky-lau**
* Add parameter for reflow on focus mixin
        - applied on links as this seems to be where outline appears incorrectly
        - related to 13.1.2
        
* Fix `:focus` being applied twice on `a`

### CODE REVIEW

File: context/brand-context/default/scss/30-mixins/focus.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
-@mixin u-focus-outline {
+@mixin u-focus-outline($reflow: false) {
 	&:focus {
 		outline: 3px solid $context--focus-color;
-		-webkit-transform: translateZ(0); // reflow needed to ensure outline styles visible in Safari v13+
+		@if $reflow == true {
+			-webkit-transform: translateZ(0); // reflow needed to ensure outline styles visible in Safari v13+
```
Comment (alexkilgour) [inline context/brand-context/default/scss/30-mixins/focus.scss:5]: Is there a negative effect of applying this all browsers? i.e. just using `transform: translateZ(0)`

Comment (ricky-lau) [reply context/brand-context/default/scss/30-mixins/focus.scss]: Replaced the `transform: translateZ(0)` hack with `will-change: transform` to create new layer. 

Removed the parameter and applied it to all browsers as I couldn't see significant effects on the page when tabbing through the page; the tabbing isn't a smooth transition that can be noticed.

File: context/brand-context/default/scss/30-mixins/focus.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
-@mixin u-focus-outline {
+@mixin u-focus-outline($reflow: false) {
```
Comment (alexkilgour) [inline context/brand-context/default/scss/30-mixins/focus.scss:1]: can remove `$reflow: false` from here?

File: context/brand-context/nature/scss/40-base/links.scss [hunk 3]
```diff
@@ -4,6 +4,7 @@
  */
 
 a {
+	@include u-focus-outline($reflow:true);
```
Comment (alexkilgour) [inline context/brand-context/nature/scss/40-base/links.scss:7]: can remove `$reflow: true` from here?

**Review by alexkilgour — APPROVED** [review]

**Pull request merged**